packages/entity-browser/src/routes/detail/components/SubGrid/SubGrid.js

```diff
@@ -9,7 +9,12 @@ const SubGrid = props => {
         entityName={props.modelField.targetEntity}
         formBase="User"
         showSearchForm={false}
-        preselectedSearchFields={[]}
+        preselectedSearchFields={[{
+          id: props.modelField.reverseRelationName,
+          value: props.entityKey,
+          hidden: true
+        }
+        ]}
```

Comment:
(I personally would merge lines 16 and 17 so that it's consistent with the opening parenthesis)